File: js/fizzbuzz.js
var fizzbuzz = function(num) {
	fizzbuzzArray = [];
	for (var i = 1; i <= num; i++) {
		if ((i % 3 == 0) && (i % 5 == 0)) {
			fizzbuzzArray.push('fizzbuzz');
		} else if (i % 3 == 0) {
			fizzbuzzArray.push('fizz');
		} else if (i % 5 == 0) {
			fizzbuzzArray.push('buzz');
		} else {
			fizzbuzzArray.push(i);
		}
	};
	return fizzbuzzArray;
}

File: rb/fizzbuzz.rb
def fizzbuzz(num)
  num_array = (1..num).to_a
  fizzbuzz_array = num_array.map do |x|
    if x % 3 == 0 && x % 5 == 0
      'fizzbuzz'
    elsif x % 5 == 0
      'buzz'
    elsif x % 3 == 0
      'fizz'
    else
      x
    end 
  end

  fizzbuzz_array
end